### DISCUSSION TITLE
Mouse Liver Antibody recommendations

### DISCUSSION THREAD

**Discussion by mickelj7** [IBEXImagingCommunity/ibex_imaging_knowledge_base#188] [General]
Hi I'm looking for Antibody combinations that have been used together in different cycles for IBEX in mice livers. I'm hoping to cut down on time validating different concentrations of antibodies that work together. I'm new to multiplex imagining, so any input on creating cycle combinations is greatly appreciated.

**Comment by anita-gola** [comment]
Hi Mickelj7, 
thanks for reaching out to the IBEXImaging Community! 
Indeed liver IBEX (specifically focused on the immune system), was performed in several papers. You can see it in [Extended Figure 5/Supp. Video 3](https://www.nature.com/articles/s41586-020-2977-2#Sec18). As well as in greater detail in Radtke et al., 2020 [MovieS5](https://www.pnas.org/doi/full/10.1073/pnas.2018488117#supplementary-materials). Assuming that you are interested in characterizing the immune landscape of the liver, Radtke et al., 2020 has a great Supplementary Table S2 which highlight the antibodies used for each cycle/clone number/dilution used. I will add that any of the panels developed for other tissues can be used/modified for liver sections assuming that the markers are expressed. If however you are not interested in immune-profiling and are looking to characterize other cell types/markers that are not included, sadly each panel will need to be validated/optimized independently de novo. However, I'd highly recomended to check out this incredible resource in the [IBEX Imaging Community ](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/reagent_resources.html), as many antibodies have been tested/validated and could be a great starting point! 
Lastly, for optimal liver imaging we perfused liver with PBS to remove all the blood, and subsequently with 2% PFA (all through the portal vein). Then the samples were fixed at 4° C for 12-16 hours in BD CytoFix/CytoPerm diluted 1:4 in PBS. Be aware that we have found that detergents (such as Triton X) can be detrimental for liver imaging, something we noticed especially in large volumes. 
Hope this is helpful!!
anita

**Reply by mickelj7** [reply]
Thank you so much for this information! We are looking at Liver zonation. 
The information about Triton X is particularly interesting. Do you have an alternative you would recommend or do you not include detergents in the blocking buffer?

**Reply by anita-gola** [reply]
We used a Saponin or Tween based detergent, such as 1X of the Perm/Wash buffer from BD.

**Reply by mickelj7** [reply]
Hello, I am see a lot of auto florescence in our first run though and I was wondering if you saw any issues like this in the liver with your samples? We are seeing the biggest issue is the 488nm and 638nm channel. 
I was also wondering if you could share what issues Triton-X caused for you in the imagining?

**Reply by radtkea** [reply]
Hi Mickelj7,

Yes, we too see auto in the 488 and 633 channels for the liver. I have included a table I created detailing the sources of autofluorescence in various tissues and potential solutions to eliminate it from our [review](https://www.nature.com/articles/s41592-021-01316-y).
<img width="809" alt="Screenshot 2024-08-15 at 9 35 03 AM" src="https://github.com/user-attachments/assets/e22d0607-6f0f-43c5-b4c3-5a98b94f46c4">

Happy to review your images and discuss the specific challenges you are facing, if needed.

For the Triton-X-100, I believe that Anita saw that certain [antibodies](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/E-cadherin_AF647/0000-0003-1431-1398.html) no longer label the tissue if this detergent is included in blocking and labeling buffers.
